### ISSUE TITLE
TipTap Target issue - Accessibility issue

### ISSUE THREAD

**Issue by jonroberts24** [umbraco/Umbraco-CMS#21574]
### Which Umbraco version are you using?

Umbraco 17.1.0

### Bug summary

Hi - I was wondering if you could help.
Our client is in the progress of upgrading from 13 to 17:
Using Umbraco 17 and TipTap as a RTE. If we dont set a Target in a link TipTap adds in a empty Target attribute to the a link - target="" - this fails accessibility rules so how can I remove target attribute if it is not set with a value - obviously this must be a TipTap setting or a TypeScript extension? I have tried adding a TS extension but this does NOT do anything. I was wondering if your developers could share a working solution please.

### Steps to reproduce

Add in a link - Save > view markup - the target="" is visible in the a tag

**Comment by leekelleher** [comment]
Fixed in #21572 for Umbraco v17.3.0.
